src/components/Auth/AuthHookForm.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import { LinkbraryIcon } from "@/src/asset";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import InputField from "./InputField/InputField";
+import styles from "./AuthHookForm.module.scss";
+
+export default function AuthHookForm() {
+  const { handleSubmit } = useForm();
+  // register : value 관련
+  // watch : input 상태감지
+  // formState : submit error 감지
+
+  return (
+    <main className={styles.authForm}>
+      <header className={styles.header}>
+        <Link href={"/"}>
+          <LinkbraryIcon width="212" height="38" />
+        </Link>
+        <div className={styles.header__text}>
+          <p>회원이 아니신가요?</p>
+          <Link href={"/signup"}>회원 가입하기</Link>
+        </div>
+      </header>
+      <section>
+        <form>
+          <InputField type="email" name="이메일" label={true} />
```

Comment:
먼저 label이 boolean 데이터 타입이라는게 어색하게 느껴지는 포인트인 것 같아요!
보통 label은 명사로 생각할 것 같습니다~
```suggestion
          <InputField type="email" name="이메일" label={true} />
```

Reply:
컴포넌트에 props로 값을 전달할 때 true값을 전달하는 경우에는 보통 이렇게 표현해줍니다~
```suggestion
          <InputField type="email" name="이메일" label />
```